staging/src/k8s.io/apimachinery/pkg/types/uid.go

```diff
@@ -20,3 +20,9 @@ package types
 // don't ONLY use UUIDs, this is an alias to string.  Being a type captures
 // intent and helps make sure that UIDs and names do not get conflated.
 type UID string
+
+// A pod UID which has been translated/resolved to the representation known to kubelets.
+type ResolvedPodUID UID
```

Comment:
If these are not accessed outside of the kubelet they don't belong in this package.  Would recommend kubelet types.